docs/css/react.scss

```diff
@@ -260,6 +260,22 @@ h1, h2, h3, h4, h5, h6 {
     &.active {
       color: $primary;
     }
+
+    &[href*="http"] {
```

Comment:
@zpao Sure, Updated. If the plugin sees an href attribute, it'll add an "external" class.